src/App.vue

```diff
@@ -14,6 +14,7 @@ v-app.app
     girder-auth(
         :register="true",
         :oauth="true",
+        :key="girderRest.token",
```

Comment:
What does this do? I've only seen `key` used with `v-for`

Reply:
It's just a generic way to control component lifecycle.  It's used with v-for as an efficient way of preserving elements through a re-render of the loop.

If key changes, the component is signaled to destroy and create a new instance of itself.   For sensitive components like login, seemed like the best way to clean up after itself.

Reply:
Cool, good to know.